docs/ref/request-response.txt

```diff
@@ -85,6 +85,15 @@ All attributes should be considered read-only, unless stated otherwise below.
     from ``GET`` or ``POST``) will use the new ``encoding`` value.  Useful if
     you know the form data is not in the :setting:`DEFAULT_CHARSET` encoding.
 
+.. attribute:: HttpRequest.content_type
+
```

Comment:
Add: `.. versionadded:: 1.10` for the new attributes.
It seems they should also be initialized in `HttpRequest.__init__()` as the other attributes that are set by `WSGIRequest` are.